.changes/next-release/feature-6597c5eb-9bb5-4f22-a555-7cd9b14be7be.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Amazon Q: Add command to generate unit tests for selected code"
```

Comment:
the changelog is confusing because this will only be available for internal users

Reply:
Changelog omitted given this is only available to internal users.